templates/defaults/vcl_recv.vcl.tmpl

```diff
@@ -57,4 +57,19 @@ sub vcl_recv {
     }
     {{ end }}
     {{ end }}
+
+    ### Pagespeed
+    # Based on the suggestions https://www.modpagespeed.com/doc/downstream-caching
+    {{ if getenv "VARNISH_PAGESPEED_SECRET_KEY" }}
+    # Tell PageSpeed not to use optimizations specific to this request.
+    set req.http.PS-CapabilityList = "fully general optimizations only";
+
+    # Don't allow external entities to force beaconing.
+    unset req.http.PS-ShouldBeacon;
+
+    if (req.method == "PURGE") {
```

Comment:
We don't need these purge related lines at all here

Reply:
OK removed.